app/src/main/res/values-hi/strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="account_balance">खाते में शेष</string>
     <string name="total_deposits">कुल जमा</string>
     <string name="open_drawer">Open Drawer</string>
-    <string name="close_drawer">Close Drawer</string>
+    <string name="close_drawer">दराज को बंद करें</string>
```

Comment:
This should not be translated, translate=false

Reply:
Sorry I did that by mistake. I corrected it now.